these fail with new libpcre, removed upstream in a different patch that doesn't apply
https://github.com/fish-shell/fish-shell/commit/ec8844d834cc9fe626e9fc326c6f5410341d532a

---
--- a/src/fish_tests.cpp
+++ b/src/fish_tests.cpp
@@ -5723,8 +5723,6 @@
         {{L"string", L"match", L"-r", L"-a", L"a*", L"b", 0}, STATUS_CMD_OK, L"\n\n"},
         {{L"string", L"match", L"-r", L"foo\\Kbar", L"foobar", 0}, STATUS_CMD_OK, L"bar\n"},
         {{L"string", L"match", L"-r", L"(foo)\\Kbar", L"foobar", 0}, STATUS_CMD_OK, L"bar\nfoo\n"},
-        {{L"string", L"match", L"-r", L"(?=ab\\K)", L"ab", 0}, STATUS_CMD_OK, L"\n"},
-        {{L"string", L"match", L"-r", L"(?=ab\\K)..(?=cd\\K)", L"abcd", 0}, STATUS_CMD_OK, L"\n"},
 
         {{L"string", L"replace", 0}, STATUS_INVALID_ARGS, L""},
         {{L"string", L"replace", L"", 0}, STATUS_INVALID_ARGS, L""},
